src/app/lib/logger.node.js

```diff
@@ -70,7 +70,12 @@ const fileLogger = createLogger({
   ),
   transports: [
     new transports.File(loggerOptions.file),
-    new transports.Console(loggerOptions.console),
+
+    // console output is sent to syslog - this can consume a lot of disk space in instances that
+    // handle a lot of traffic, so we only enable console output in some environments
+    ...(process.env.LOG_TO_CONSOLE === 'true'
```

Comment:
Will we ever want this in Test / Live? If not, would it just make sense to use `process.env.APP_ENV === 'local'` instead?

Reply:
That would mean we were not able to see the console logs on localhost though right? Thats something we do not want to loose

Reply:
Yup - see console output in local environment only. Just wondering if we need a separate env variable for it, or if we can just use the APP_ENV variable instead.

Reply:
Deva may want to see logs on live env being run locally - wanted to make it easy to enable that in case it is needed temporily

Reply:
:shipit: